test/functional/Valhalla/testng.xml

```diff
@@ -47,7 +47,7 @@
 	</test>
 	<test name="NullRestrictedTypeOptTests">
 		<classes>
-			<class name="org.openj9.test.lworld.NullRestrictedTypeOptTests"/>
+			<!--<class name="org.openj9.test.lworld.NullRestrictedTypeOptTests"/>-->
```

Comment:
I guess there will be more tests to be temporarily disabled ? If yes, it might be worth creating an issue to track these disabled tests and include the issue number here.

Reply:
Or you can use the existing issue https://github.com/eclipse-openj9/openj9/issues/18157 to list and track the disabled tests.

Reply:
I think both makes sense, we'll just have to be mindful of making sure any new tests get reflected in each set.

Reply:
> should we keep the old versions of these tests alive while keywords like primitive and the Q signature prefix still work?

Looks safer to me that we keep the old version alive.

Reply:
Okay I'll modify my to keep both.

Reply:
I changed it so there is one version of NullRestrictedTypeOptTests in src_lw5 and one in src_qtypes with the idea that you could enable one or the other as needed. Of course src_lw5 doesn't work completely yet since there are many other tests that need updating. Let me know what you think.

Reply:
I think your change this looks good.